lib/utils.js

```diff
@@ -615,11 +613,20 @@ exports.constants = exports.defineConstants({
   MOCHA_ID_PROP_NAME
 });
 
+const uniqueIDBase =
+  'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_';
+
 /**
  * Creates a new unique identifier
  * @returns {string} Unique identifier
  */
-exports.uniqueID = () => nanoid();
+exports.uniqueID = () => {
+  let id = '';
+  for (let i = 0; i < 21; i++) {
+    id += uniqueIDBase[(Math.random() * 64) | 0];
+  }
+  return id;
+};
```

Comment:
Can we mention in the comment that this is essentially a copy of `nanoid/non-secure`?

Especially as that explains why `Math.random` is okay whereas it typically shouldn't be used for unique ID:s (I wonder where in the code this is used?)